climada/hazard/tc_tracks.py

```diff
@@ -396,11 +488,7 @@ def read_ibtracs_netcdf(self, provider=None, storm_id=None,
         ibtracs_ds['id_no'] = (ibtracs_ds.sid.str.replace(b'N', b'0')
                                .str.replace(b'S', b'1')
                                .astype(float))
-        ibtracs_ds['time_step'] = xr.zeros_like(ibtracs_ds.time, dtype=float)
-        ibtracs_ds['time_step'][:, 1:] = (ibtracs_ds.time.diff(dim="date_time")
-                                          / np.timedelta64(1, 'h'))
-        ibtracs_ds['time_step'][:, 0] = ibtracs_ds.time_step[:, 1]
-        provider = provider if provider else 'ibtracs'
+        provider_str = f"ibtracs_{provider[0]}" + ("" if len(provider) == 1 else "_mixed")
```

Comment:
`provider_str` does not allow to unambiguously know what the `provider` input was. Why not append all provider elements together? Not strictly needed and anyway not used elsewhere in CLIMADA I presume, but what do you think?

Reply:
I thought about that before, but didn't want the string to become too long because people might use it in file names or in database entries.